src/aggregationserver/aggregationbroker/aggregationreplier.cpp

```diff
@@ -133,6 +133,38 @@ AggServer::AggregationReplier::ProcessExperimentStateRequest(const AggServer::Ex
     const int run_id = message.experiment_run_id();
 
     try {
+        const auto& run_results = database_->GetValues(
+            "ExperimentRun",
+            {
+                "JobNum",
+                "to_char((starttime::timestamp), 'YYYY-MM-DD\"T\"HH24:MI:SS.US\"Z\"') AS StartTime",
```

Comment:
Maybe move this string creation into a function and/or statically allocate this format string.

Reply:
Addressed in [](https://github.com/cdit-ma/logan/pull/40/commits/800822112b466d82392478c733823cee7a0d9935)